apps/tutorial-servo-can-control/include/TutorialServo.h

```diff
@@ -0,0 +1,37 @@
+#ifndef MBED_TUTORIALSERVO_H
+#define MBED_TUTORIALSERVO_H

```

Comment:
you don't need this define guard since you already have `pragma once` which does the same thing